internal/genesis/foundational.go

```diff
@@ -2,8 +2,14 @@ package genesis
 
 // GenesisFNAccounts are the ECSDA accounts for the foundational nodes.
 var GenesisFNAccounts = [...]DeployAccount{
+	// st
```

Comment:
Please remove the account owner comments; we omitted them from this file for security reasons.

Please also put new entries at the end; otherwise everyone's sharding assignment may shift and result in questions (“hey my shard changed; should I be worried?”)